### PR TITLE
Update all dependencies and fix new linter errors

### PR DESCRIPTION
**Pull request by sethvargo**
/cc @bharathkkb

### CODE REVIEW

File: .github/workflows/setup-cloud-sdk.yml [hunk 1]
```diff
@@ -1,37 +1,51 @@
...
+        - '14'
     steps:
-    - uses: actions/checkout@v2
+    - uses: 'actions/checkout@v2'
 
-    - uses: actions/setup-node@master
+    - uses: 'actions/setup-node@v2'
       with:
-        node-version: 12.x
+        node-version: '${{ matrix.node }}.x'
```
Comment (sethvargo) [inline .github/workflows/setup-cloud-sdk.yml:35]: I think this was a CI bug - we were never actually setting the node.js version (we were running node 12 for all tests)

**Review by sethvargo — COMMENTED** [review]
@bharathkkb do you know if the `dist/*.d.ts` files are required? I couldn't figure out how they were generated.

**Review by averikitsch — APPROVED** [review]

**Comment by averikitsch** [comment]
> @bharathkkb do you know if the `dist/*.d.ts` files are required? I couldn't figure out how they were generated.

This is generated on `npm run build`. I believe I've kept it so we can have both Typescript and Javascript library support.

**Comment by sethvargo** [comment]
> This is generated on npm run build. I believe I've kept it so we can have both Typescript and Javascript library support.

@averikitsch - I can't seem to get it to generate with `npm run build`. That only generates the index.js file

**Comment by sethvargo** [comment]
Nvm - figured it out 😄 . Needed to fix the tsconfig

**Pull request merged**